packages/terra-dev-site/CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 ## Unreleased
 
+## 8.1.0 - (June 7, 2022)
+
 * Changed
-  * updated `terra-application` version in peer dependency.
```

Comment:
Is the removal if this line intentional?

Reply:
Yes. The original line didn't express the new feature (IE 10 support) which justified the minor version bump. So, once I determined this new feature would be pulled in I edited the line to make it more descriptive to our consumers.

If, on the other hand, I'm wrong about IE 10 date-time functions being _used_ in terra-dev-site then I can revert this and instead bump this package as a patch (8.0.1).